core-services/template-deprecation/_allowlist.yaml

```diff
@@ -2095,6 +2095,9 @@ templates:
         periodic-ci-operator-framework-operator-lifecycle-managment-rhoperator-metric-e2e-aws-olm-release-4.5-daily:
           generated: false
           kind: periodic
+        pull-ci-integr8ly-integreatly-operator-rhoam-release-v1.1-rhoam-e2e:
```

Comment:
Why this job can't run with multi-stage steps implementation?

Reply:
/hold

Reply:
@droslean this was generated to fix the allowlist test failure. I'm not exactly sure what the configuration is about.

Reply:
If you could advise me on what the configuration here should be that would be great. I can remove this entry if it's not appropriate.

Reply:
@laurafitzgerald This [test](https://github.com/openshift/release/pull/14818/files#diff-bf8113af8e90450ace44f3700f59c7f24df8de2f6e3111c98c1bd309619b12e8R71-R74) is using a template which we don't allow it except if you put the job in the allowlist. My question is why we need to allow this test to run with a template and not with a multi-stage implementation? What is the blocker here?

Reply:
Please check https://docs.ci.openshift.org/docs/getting-started/examples/